josh-core/src/query.rs

```diff
@@ -162,13 +193,25 @@ pub fn render(
     drop(obj);
     drop(tree);
 
+    let repo_path =
+        if transaction.repo().path().file_name() == Some(std::ffi::OsStr::new("overlay")) {
```

Comment:
can that be a problem if the repo is really called overlay? (I guess that will never be the case with our docker deployement)

Reply:
That's a good point 🤔 
This will be a problem when using the cli on a repo called "overlay" so not so good. I'll find a different way of doing this.

Reply:
Done